assets/templates/partials/filter.tmpl

```diff
@@ -1,83 +1,94 @@
-{{ $filter := .Data.Filter }}
+{{ $filters := .Data.Filters }}
 {{ $response := .Data.Response }}
 {{ $query := .Data.Query }}
+{{ $sort := .Data.Sort.Query }}
 {{ $lang := .Language }}
-<form id="filterForm" class="js-auto-submit__form" action="/search" method="get">
+<form id="filterForm" method="get">
```

Comment:
Think this change may break non-js version as it will no longer submit to server?